scripts/spec/newsSpec.js

```diff
@@ -0,0 +1,11 @@
+it ("news should have a title, url, thumbnail and summary", () => {
+  let news = new News();
+  news.title = "today's news";
+  news.url = "http://news.com";
+  news.thumbnail = "http://news.jpg";
+  news.summary = "this is today's news";
+  expect(news.title).toEqual("today's news")
+  expect(news.url).toEqual("http://news.com")
+  expect(news.thumbnail).toEqual("http://news.jpg")
+  expect(news.summary).toEqual("this is today's news")
+})
```

Comment:
I wonder whether this could possibly be broken down into multiple it/expects

Reply:
This test doesn't really test anything of your code. You set a variable to something, and then test that that variable's value is something. This code above would work even if the News class was completely empty.
The good news is that testing state is not really useful and you can just remove this test.